src/wp_ecx_kmgmt.c

```diff
@@ -1540,7 +1543,7 @@ static int wp_ed25519_import_private(const byte* in, word32 inLen,
 /** Ed25519 data and wolfSSL functions. */
 static const wp_EcxData ed25519Data = {
     WP_KEY_TYPE_ED25519,
-    255,
+    256,
```

Comment:
This is no longer bits in curve.
Change to comment on the field in the struct definition.